src/GrommetSwagger.js

```diff
@@ -60,14 +61,17 @@ export default class GrommetSwagger extends Component {
       .then(response => response.text())
       .then(text => jsyaml.load(text))
       .then(filterHiddenPaths)
-      .then((data) => {
-        document.title = data.info.title;
+      .then(data => SwaggerParser.validate(data)
```

Comment:
I'm pretty sure you tested against this but does [this line](https://github.com/HewlettPackard/hpe-onesphere-http/blob/master/swagger.yml#L3568) validate in the the parser?